admin/securedrop_admin/__init__.py

```diff
@@ -1183,8 +1200,8 @@ class ArgParseFormatterCombo(
     parse_install = subparsers.add_parser("install", help=install_securedrop.__doc__)
     parse_install.set_defaults(func=install_securedrop)
 
-    parse_tailsconfig = subparsers.add_parser("tailsconfig", help=run_tails_config.__doc__)
-    parse_tailsconfig.set_defaults(func=run_tails_config)
+    parse_localconfig = subparsers.add_parser("localconfig", help=run_local_config.__doc__)
```

Comment:
Apparently subparsers can have aliases, so we could add an alias for `tailsconfig` and put off the docs change until the rest of the Qubes admin work (packaging etc) is done with eg:
```
parse_localconfig = subparsers.add_parser("localconfig", aliases =['tailsconfig'], help=run_local_config.__doc__)

Reply:
Smart. I'm adding both `tailsconfig` and `qubesconfig` as aliases.